src/test/java/uk/gov/hmcts/reform/sscs/model/SessionCategoryMapsTest.java

```diff
@@ -0,0 +1,99 @@
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategorySecondDoctor() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("064", "GC", true, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.INDUSTRIAL_DEATH_BENEFIT);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.GC);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_06);
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code and no secondDoctor is given to getHearingDuration"
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategoryNoSecondDoctor() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("064", "GC", false, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.INDUSTRIAL_DEATH_BENEFIT);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.GC);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_05);
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code but secondDoctor shouldn't be given to getHearingDuration"
+            + "null is returned")
+    @Test
+    public void getSessionCategorySecondDoctorInvalid() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("003", "LE", true, false);
+
+        assertThat(sessionCategory).isNull();
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code and a secondDoctor is given to getHearingDuration"
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategoryFqpmRequired() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("023", "RB", false, true);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.CHILD_SUPPORT_REFORMS);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.RB);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_02);
+    }
+
+    @DisplayName("When valid Benefit Code, Issue Code and no secondDoctor is given to getHearingDuration"
+            + "the valid mapping is returned")
+    @Test
+    public void getSessionCategoryFqpmNotRequired() {
+        SessionCategoryMap sessionCategory = sessionCategoryMaps
+                .getSessionCategory("023", "RB", false, false);
+
+        assertThat(sessionCategory).isNotNull();
+        assertThat(sessionCategory.getBenefitCode()).isEqualTo(BenefitCode.CHILD_SUPPORT_REFORMS);
+        assertThat(sessionCategory.getIssue()).isEqualTo(Issue.RB);
+        assertThat(sessionCategory.getCategory()).isEqualTo(SessionCategory.CATEGORY_01);
+    }

```

Comment:
As above!

Reply:
The test is for SessionCategory, so this is already doing that